wrapper/src/test/java/integration/container/aurora/mysql/mariadbdriver/AuroraMysqlIntegrationTest.java

```diff
@@ -478,7 +478,8 @@ public void test_failoverBackToThePreviouslyDownReader()
 
       assertTrue(
           firstReaderInstanceId.equals(lastInstanceId)
-              || secondReaderInstanceId.equals(lastInstanceId));
+              || secondReaderInstanceId.equals(lastInstanceId)
+              || writerInstanceId.equals(lastInstanceId));
```

Comment:
Since this test's comments mention the purpose of the test is to test the ability to revive previously down reader instances, should we crash the writer instance before the last assertFirstQueryThrows instead of adding this line here?

Reply:
I personally prefer updating the test name and include this behaviour in the test.
What do you think @sergiyvamz

Reply:
I'd agree with @karenc-bq and change the test name.